projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -226,6 +232,8 @@ describe('ClrAccordionPanel', () => {
       expect(headerButton.getAttribute('aria-controls')).toBe(panelRegion.getAttribute('id'));
       expect(headerButton.getAttribute('disabled')).toBe(null);
 
+      expect(panelHeading).toBeNull();
```

Comment:
This should be extracted to a test specifically for the feature flag not being enabled. Also, there should be tests for the feature flag being enabled without an explicit heading level value for a parent and nested accordion. Same for the stepper.

Reply:
This is redundant now, so I think it should be removed.

Reply:
Corrwcted in commit 2cb2400